src/config/resolve-config.js

```diff
@@ -47,8 +48,8 @@ const getExplorerMemoized = mem(opts => {
     }
   });
 
-  const load = opts.sync ? explorer.loadSync : explorer.load;
-  const search = opts.sync ? explorer.searchSync : explorer.search;
+  const load = explorer.load;
+  const search = explorer.search;
```

Comment:
We can simplify this - `const { load, search } = explorer;`

Reply:
sure

Reply:
we don't even need this const, use `explorer.load` directly